app/views/ssh_keys/_ssh_keys_tab.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="tab-pane" id="ssh_keys">
+  <table class="<%= table_css_classes 'table-fixed' %>">
+    <% if user.ssh_keys.any? %>
+    <thead>
+      <tr>
+        <th width="40%"><%= _('Name') %></th>
```

Comment:
Please use `class="col-md-x col-sm-x"` instead of fixed percentages and pixels - it's way more flexible and looks good on smaller screens